ConsentViewController/Classes/LocalStorage/SPUserDefaults.swift

```diff
@@ -17,6 +17,9 @@ class SPUserDefaults: SPLocalStorage {
     static let USER_DATA_KEY = "\(SP_KEY_PREFIX)userData"
     static let IAB_CMP_SDK_ID_KEY = "\(IAB_KEY_PREFIX)CmpSdkID"
     static let IAB_CMP_SDK_ID = 6
+    private let GDPR = "GDPR"
```

Comment:
@Nevazhnovu what about:
```swift
static let GPDR_CHILD_PM_ID_KEY = "\(SPUserDefaults.SP_KEY_PREFIX)\(GDPR)childPmId"
```
This way you can just use this value when getting/setting the value from storage, instead of "building" the key in the getter/setters.

Same is valid for the CCPA version.